lib/TerminalDisplay.h

```diff
@@ -62,6 +62,13 @@ namespace Konsole
         MoveEndScreenWindow = 2
     };
 
+    enum BackgroundMode {
```

Comment:
See the `enum` above it -- also the same for the place. I think, if a change is really needed, it could be made for both and in a separate patch.

Reply:
> if a change is really needed, it could be made for both and in a separate patch.

That makes sense. I will create an issue after this PR is merged.

Reply:
Thanks! You have a wider perspective on this code, while I'm a passer-by who may miss the wood for the trees.